bind9/README.md

```diff
@@ -1,77 +1,229 @@
-# Bind9 check Integration
-
 ## Overview
 
-Get metrics from Bind9 DNS Server.
+[Bind 9][19] is a complete, highly portable implementation of the Domain Name System (DNS) protocol. The Bind9 name server (named), can act as an authoritative name server, recursive resolver, DNS forwarder, or all three simultaneously.
```

Comment:
The correct name seems to be "BIND 9", but I know this integration already exists, so maybe just replace instances of "Bind9" with "Bind 9" to keep it consistent within the integration docs? This may not be possible since some of the configuration instructions reference "Bind9" and that might be important to keep. Regardless, I recommend making all the references consistent.

Reply:
Replaced instances of "Bind9" to "Bind 9" apart from the configuration steps where it's not possible to update.